px-data-grid-header-cell.html

```diff
@@ -142,6 +158,12 @@
           if (!dropdownOpened && !this._mouseover) {
             this._dropdownHidden = true;
           }
+
+          if (dropdownOpened) {
+            this.parentElement.assignedSlot.parentElement.style.zIndex = '3';
```

Comment:
1. DRY: assign the variable before if statement
2. add the explanatory comment why this is needed

Reply:
Done.